src/main/java/io/jenkins/plugins/extratoolinstallers/installers/IsAlreadyOnPath.java

```diff
@@ -356,7 +118,7 @@ public FormValidation doCheckVersionCmdString(@QueryParameter String value) {
         }
```

Comment:
More refactoring needed (sorry).
The only methods in this `DescriptorImpl` class should be those that are unique to the `IsAlreadyOnPath` class.
i.e. validating all the fields that are in `VersionCheckingToolInstaller` is something that should be done by code within a `DescriptorImplBase` class within the `VersionCheckingToolInstaller.java` code.
...and the resources should also reflect the same hierarchy too.

e.g. if you've got a abstract base class `Foo` that defines a field `bar` then Foo should provide the getter/setter code _and_ the `public FormValidation doCheckBar(@QueryParameter String value)` method.
...and the `help-bar.html` file should be in `resources/.../Foo/` too.